packages/foam-vscode/src/features/create-from-template.ts

```diff
@@ -407,29 +459,64 @@ async function createNoteFromTemplate(
     resolvedValues,
     templateMetadata
   );
-  const defaultFilename = path.basename(defaultFilepath.path);
 
   const filepath = await askUserForFilepathConfirmation(
-    defaultFilepath,
-    defaultFilename
+    defaultFilepath.fsPath,
+    path.basename(defaultFilepath.path)
   );
 
   if (filepath === undefined) {
     return;
   }
-  const filepathURI = Uri.file(filepath);
-  await writeTemplate(templateSnippet, filepathURI);
+  const filepathUri = Uri.file(filepath);
+  await writeTemplate(templateSnippet, filepathUri);
+}
+
+function askTemplateCreationLocation(): Thenable<QuickPickItem | undefined> {
+  const homeDirQuickPick = {
+    label: 'In your home directory',
```

Comment:
I like that you show the user where to create the template. A couple of things:
- to make things as simple as possible for the user, should we nudge them in a direction? 
I would say workspace template as it feels like the more standard case, but open to other opinions.

- what about the terminology? I would be keen to focus on functionality (global vs local/workspace templates) vs implementation (that global templates live in the user's directory).  
Not sure whether we should say `workspace` vs `global` vs `device`. We want to convey that `workspace` templates are local, but also portable with the workspace. `global`/`device` templates can be reused on the same machine, but do live on the machine (well, actually on the user, so maybe we should use `user` and mimic VS Code terminology: `Workspace` and `User`)